docs/application/dotnet/guides/nui/nui-components/CameraView.md

```diff
@@ -0,0 +1,62 @@
+---
+keyword: camera, CameraView, NUI, CameraView.DisplayType
+---
+
+# CameraView
+
+The [CameraView](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.BaseComponents.CameraView.html) class is the NUI view that displays [camera](../../multimedia/camera.md).
+
+
+## Create a CameraView
+
+The following basic example shows how to create a `CameraView` object:

```

Comment:
Global comment: 

Kindly update CameraView as `CameraView` at all other places in this topic.